driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -941,6 +978,10 @@ public void run() {
             return ccm;
         }
 
+        private static boolean isThriftSupported(boolean dse, VersionNumber version) {
+            return dse || version.compareTo(VersionNumber.parse("4.0")) < 0;
```

Comment:
Just curious, but what is the significance of DSE here?  Although all versions of DSE support Thrift at the moment, i imagine that will change too.  That being said I think it is fine this way for now.